components/payment-term.jsx

```diff
@@ -161,33 +186,16 @@ export function PaymentTerm({
 
 	return (
 		<div id={fieldId} className="o-forms__group ncf__payment-term">
-			{options.map((option) => createPaymentTerm(option))}
+			<div
+				className={`${optionsInARow ? 'ncf__payment-term__options-grid' : ''}`}
```

Comment:
Do we need the ` here?  Shouldn't this work as well and is easier to read?
```suggestion
				className={optionsInARow ? 'ncf__payment-term__options-grid' : ''}
```